Marlin/Configuration.h

```diff
@@ -3171,7 +3171,8 @@
     //#define SINGLE_TOUCH_NAVIGATION
   #endif
 
-//#define TOUCH_USE_ENCODER_TOO         // Use encoder or buttons control together with touchscreen
+//#define DISABLE_ENCODER         // Don't use encoder or buttons control together with touchscreen,
```

Comment:
This should be moved under`#if ENABLED(TOUCH_SCREEN)` section above to ensure that it only applies to touchscreen-based configs & will be clearer to to users that it's not a general option to disable the encoder.